### PR TITLE
Update Figma API typings to version 9

### PR DESCRIPTION
**Pull request by rudi-c**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://www.figma.com/plugin-docs/api/typings/
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/figma/index.d.ts [hunk 1]
```diff
@@ -713,3 +808,6 @@ interface Image {
     readonly hash: string;
     getBytesAsync(): Promise<Uint8Array>;
 }
+} // declare global
+
+export {};
```
Comment (andrewbranch) [inline types/figma/index.d.ts]: Just curious, why the change to a global-augmenting module from the global declarations?

Comment (rudi-c) [reply types/figma/index.d.ts]: The global `figma` variable was no longer being automatically picked up by `tsc` and needed an import statement, this happened to fix it.

Comment (andrewbranch) [reply types/figma/index.d.ts]: I would think that _after_ this change, it would require an `import 'figma'` to get the globals included. If you’re trying to get the globals to show up without an import declaration, can you send me a repro of it not being picked up so I can dig into that?
